build/ci/evergreen.yml

```
@@ -2067,13 +2067,14 @@ buildvariants:
       <<: *go_linux_version
     tasks:
       - name: ".coverage"
-  - name: ubuntu
+  - name: snyk
     display_name: Snyk
+    allowed_requesters: ["patch", "ad_hoc", "github_pr"]
     tags:
-    - snyk
-    - cron
+      - snyk
+      - cron
     run_on:
-    - ubuntu2204-large
+      - ubuntu2204-large
```

any reason why we need an ubuntu large to run snyk? can't we use a small one instead? you can find more info about the available distro here:  https://spruce.mongodb.com/image/ubuntu2204/build-information#distros
```suggestion
      - ubuntu2204-small
```

---

No reason, I'll update it +1